src/status_im/contexts/wallet/account/tabs/view.cljs

```diff
@@ -7,18 +7,20 @@
     [status-im.contexts.wallet.common.activity-tab.view :as activity]
     [status-im.contexts.wallet.common.collectibles-tab.view :as collectibles]
     [status-im.contexts.wallet.common.empty-tab.view :as empty-tab]
-    [utils.i18n :as i18n]))
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
 
 (defn view
   [{:keys [selected-tab]}]
-  [rn/view {:style {:flex 1}}
-   (case selected-tab
-     :assets       [assets/view]
-     :collectibles [collectibles/view]
-     :activity     [activity/view]
-     :permissions  [empty-tab/view
-                    {:title        (i18n/label :t/no-permissions)
-                     :description  (i18n/label :t/no-collectibles-description)
-                     :placeholder? true}]
-     :dapps        [dapps/view]
-     [about/view])])
+  (let [current-viewing-account (rf/sub [:wallet/current-viewing-account-address])]
```

Comment:
I believe we can move this subscription to `collectibles/view` as this view doesn't depend on it. WDYT?

Reply:
Agree on this, I think it a more elegant solution, unless there is any other valid reason

Reply:
@smohamedjavid  @briansztamfater  Thanks for your feedback! 

I found cleaner to pass the key `:address` to the components to say "show this component for a specific address" and when not passed, just show for all addresses.

I knew this was difficult to decide, I implemented both and at the end decided to keep this because I thought we could use this aproach for the remaning components.

Now that I got you feedback I see it different, I'll do the check inside the component! :+1:

Reply:
Done